Automattic/jetpack
# Docker: silence errors for not defined constants on uninstall

When running `jetpack docker uninstall`, the script will also attempt to remove all multisite-related constants from wp-config.php. Since these constants are not defined at all on single 
site installs, `wp config delete` will output errors. While we echo a note to ignore the errors, let's try to get rid of them.

## Proposed changes:

* Pipe the output of `wp config delete` to grep to remove strings with "not defined" and ` || true` just to keep a 0 status code (not that this matters at this time)

### Other information:


## Jetpack product discussion


n/a

## Does this pull request change what data or activity we track or use?


no

## Testing instructions:






* Before patch, `jetpack docker up -d && jetpack docker install`
* `jetpack docker uninstall` -- see the errors.
* Apply patch and rerun `jetpack docker install`
* `jetpack docker uninstall` -- no errors
* `jetpack docker install && jetpack docker multisite-convert` -- reinstall and convert to multisite
* `jetpack docker uninstall` -- confirm success messages are displayed.
